tcms/core/tests/test_admin.py

```diff
@@ -8,7 +8,6 @@
 
 from tcms.tests import BasePlanCase, BaseCaseRun
 from tcms.tests.factories import UserFactory
-from tcms.tests.factories import TestCaseEmailSettingsFactory
 
 
 class TestAdminView(BasePlanCase):
```

Comment:
ATM I don't see a reason why we should inherit from BasePlanCase. Maybe just LoggedInTestCase, no ?